cmd/operator/main.go

```diff
@@ -88,6 +88,7 @@ func main() {
 		evaluatorMemoryResource = flag.Int64("evaluator-memory-resource", 200, "The Memory Resource of evaluator pod, in mega bytes.")
 		evaluatorMemoryLimit    = flag.Int64("evaluator-memory-limit", 1000, "The Memory Limit of evaluator pod, in mega bytesv.")
 		evaluatorCPUResource    = flag.Int64("evaluator-cpu-resource", 100, "The CPU Resource of evaluator pod, in milli cpu.")
+		operatorEnv = flag.String("opeator-env", "prod", "the environment operator runs on.")
```

Comment:
Instead of `prod`, aren't we more interested in:
- kubectl-managed
- gcloud-managed on GKE or GKE autopilot
- self-deployed
Or is this flag for a different purpose?

Reply:
done.